doc/CHANGES.md

```diff
@@ -2,18 +2,30 @@
 
 ## Highlighted Features
 
+* This release rolls out Alaveteli Pro Pricing - a way to ask users to support
+  your site by paying for access to the advanced Alaveteli Pro features?
+  (Liz Conlan, Gareth Rees, Graeme Porteous)
 * Add new rake task to create the Stripe webhook endpoint (Liz Conlan)
 * Send weekly metrics email to the Pro Admin team (Liz Conlan, Gareth Rees)
 * Improve error handling when sending request-related emails (initial request
   mails and followups) - failed messages are captured and a send_error event is
   logged to make it easier for site admins to see what's happened (Nigel Jones)
 * Fixed bug where expiring embargoes were not fully removed from batches when
   the related requests reached the publication data (Liz Conlan)
+* Send daily webhook digest to the Pro Admin team (Graeme Porteous)
 
 ## Upgrade Notes
 
+* You can run this release without using the Alaveteli Pro Pricing functionality
+  - by default it is switched off.
+* We have specified a Stripe API version of `2017-01-27` if you are running Pro
+  with pricing enabled you should consider re-creating the Stripe Webhook using
+  the new `bundle exec rails stripe:create_webhook_endpoint` to ensure the
+  versions are consistent
```

Comment:
Will also need to delete the old webhook afterwards otherwise they'll get duplicate webhook events

Reply:
Delete or disable?

Reply:
Good question - disabled is probably fine (less stressful for the user as the consequences of getting the wrong one is just "oh, I'll enable it again" rather than "argh! now it's gone!"). We could advise deleting disabled webhooks if it causes a problem.